src/main/java/core/basesyntax/Table.java

```diff
@@ -0,0 +1,30 @@
+package core.basesyntax;
+
+public class Table {
+    private int buy;
+    private int supply;
+    private int result;
```

Comment:
The 'result' field should be a local variable rather than a class field. It is only used to calculate a value in the setResult method, which suggests it does not need to be part of the object's state.